js/Bip39.js

```diff
@@ -0,0 +1,18 @@
+import bip39 from 'bip39';
+import { Buffer } from 'safe-buffer';
+import RustModule from './RustModule';
+import Blake2b from './Blake2b';
+import { apply } from './utils/functions';
+
+export const mnemonicToSeed = (module, mnemonic) => (
+  Blake2b.blake2b_256(bip39.mnemonicToEntropy(mnemonic))
+);
+
+export const mnemonicToEntropy = (mnemonic) => (
+  Buffer.from(bip39.mnemonicToEntropy(mnemonic), 'hex')
+);
```

Comment:
It might be worth checking how all of this can work with what is already with daedalus:

https://github.com/input-output-hk/daedalus/blob/b85ad0e212f22236d6404dd3b7c7a9fe197f410c/lib/decrypt.js

Since Daedalus already has already [a function to retrieve the entropy from a mnemonic](https://github.com/input-output-hk/daedalus/blob/b85ad0e212f22236d6404dd3b7c7a9fe197f410c/lib/decrypt.js#L33) might be worth keeping using the entropy here. and not adding this extra Bip39 handling.